specification/sql/resource-manager/Microsoft.Sql/preview/2023-05-01-preview/JobAgents.json

```diff
@@ -362,6 +416,24 @@
           }
         }
       }
+    },
+    "JobAgentUserAssignedIdentity": {
```

Comment:
>JobAgentUserAssignedIdentity [](http://example.com/codeflow?start=5&length=28)

Looks exactly like

[https://github.com/Azure/azure-rest-api-specs/blob/5dd1107d5f2be8d600325d795450e1d854fbe7e8/specification/common-types/resource-management/v5/managedidentity.json#L18C6-L18C26](https://github.com/Azure/azure-rest-api-specs/blob/5dd1107d5f2be8d600325d795450e1d854fbe7e8/specification/common-types/resource-management/v5/managedidentity.json#L18C6-L18C26)

Reply:
We would like to keep this consistent with other Microsoft.Sql APIs such as: https://github.com/Azure/azure-rest-api-specs/blob/def187e2e78d7173d8fdd7f77740dd9719e1dfbf/specification/sql/resource-manager/Microsoft.Sql/preview/2023-02-01-preview/Databases.json#L875C7-L875C7